Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -134,6 +134,11 @@
     pointSize : "5"
 });
 
+addStyle('Clamp and Mix', {
+    color : "clamp(color(), 0.0, 100.0)",
+    pointSize : "mix(0.0, 2.0, 0.5) * 5"
+});
```

Comment:
@lilleyse Also done! Also, I figured out that I can't run the Sandcastle app locally because in the Sandcastle folder, index.html refers to a few scripts that simply don't exist for me; it's easy to see using the javascript console in my chrome browser.